pkg/collector/corechecks/cluster/orchestrator/orchestrator.go

```diff
@@ -340,21 +345,43 @@ func (o *OrchestratorCheck) processNodes(sender aggregator.Sender) {
 	}
 	groupID := atomic.AddInt32(&o.groupID, 1)
 
-	messages, err := processNodesList(nodesList, groupID, o.orchestratorConfig, o.clusterID)
+	nodesMessages, clusterModel, err := processNodesList(nodesList, groupID, o.orchestratorConfig, o.clusterID)
 	if err != nil {
 		o.Warnf("Unable to process node list: %s", err) //nolint:errcheck
 		return
 	}
+	sendNodesMetadata(sender, nodesList, nodesMessages, o.clusterID)
 
+	clusterMessage, clusterErr := extractClusterMessage(o.orchestratorConfig, o.clusterID, o.apiClient, groupID, clusterModel)
+	if clusterErr != nil {
+		_ = o.Warnf("Could not collect orchestrator cluster information: %s, will still send nodes information", err)
```

Comment:
```suggestion
		o.Warnf("Could not collect orchestrator cluster information: %s, will still send nodes information", err)
```

Reply:
Do we prefer `//nolint:errcheck` over ignored return?

Reply:
ah yeah I missed that, we already do line 350, so let's stay consistent

Reply:
👍  changed everywhere to be `_=` in that file